backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -1,6 +1,39 @@
 module Spree
   module Admin
     module NavigationHelper
+      # Add items to current page breadcrumb heirarchy
+      def add_breadcrumb(*ancestors, &block)
+        breadcrumbs.concat(ancestors) if ancestors.present?
+        breadcrumbs.push(capture(&block)) if block_given?
+      end
+
+      def add_page_title_to_breadcrumbs
+        ActiveSupport::Deprecation.warn('content_for(:page_title) is deprecated, use add_breadcrumb')
+        add_breadcrumb(content_for(:page_title))
+      end
+
+      # Render Bootstrap style breadcrumbs
+      def render_breadcrumbs
+        add_page_title_to_breadcrumbs if content_for?(:page_title)
+        content_tag :ol, class: 'breadcrumb' do
+          safe_join breadcrumbs.collect { |level|
```

Comment:
Both of these are valueable comments, I'd like to see them addressed (`map` > `collect`, `do / end` for multi-line blocks).